librz/include/rz_il/definitions/value.h

```diff
@@ -0,0 +1,47 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef RZ_IL_VALUE_H
+#define RZ_IL_VALUE_H
+
+#include "variable.h"
+
+/// The following RzIL_* is only a mark
+/// to remind developer the argument type in opcode struct
+#define RzIL_PURE_VAL
+#define RzIL_BITV
+#define RzIL_BOOL
+#define RzIL_VAR
+#define RzIL_EFF
+#define RzIL_LABLE
```

Comment:
typo here. However, I think these are not necessary anymore given that we can just use the right types now instead of `void *`

Reply:
Yes